src/ensembl/webpack/webpack.common.js

```diff
@@ -31,7 +31,7 @@ module.exports = (isDev, moduleRules, plugins) => ({
       // babel-loader is used for typescript as it has good tree shaking support compared to tsc
       {
         test: /.tsx?$/,
-        loader: 'babel-loader',
+        use: ['babel-loader', 'eslint-loader'],
```

Comment:
Doesn't this mean that code with linting errors will fail to even compile and run? If so, let's not?

Reply:
Ah, that's a good question. I just checked it and turns out it compiles but does not display any changes made to the code with HMR. However, works on a reload (probably means it compiled?).  Anyway, I agree this is not ideal and I'll try to find a better alternative to this.

Reply:
Alternative: run eslint as part of pre-commit hook (with husky).

Reply:
This is now fixed by using the `emitWarning` option. ESLint will print these errors that don't break the code as warnings in the browser and the console. However, it will clearly state if they are errors or warnings.